analyzers/src/SonarAnalyzer.Common/SonarAnalyzer.Common.csproj

```diff
@@ -10,6 +10,10 @@
     <PackageReference Include="Google.Protobuf" Version="3.6.1" />
     <!-- When changing this reference update ProtocExe property too! -->
     <PackageReference Include="Google.Protobuf.Tools" Version="3.6.1" />
+    <PackageReference Include="Grpc.Tools" Version="2.26.0">
```

Comment:
Is there a reason to use 2.26 instead of the latest 2.62?

Reply:
Yes, the latest one doesn't work. Probably, we would have to update protobuf, too.  Due to this, I've used the same version we use on sonar security.